Dockerfile

```diff
@@ -1,12 +1,11 @@
-FROM alpine:edge
+FROM alpine:3.14.0
 
 WORKDIR /goaccess
 
 ARG build_deps="build-base ncurses-dev autoconf automake git gettext-dev geoip-dev wget zlib-dev bzip2-dev"
 ARG runtime_deps="tini ncurses libintl gettext openssl-dev geoip zlib libbz2"
 
 RUN apk add --no-cache -u $runtime_deps $build_deps && \
-    /etc/periodic/monthly/geoip && \
```

Comment:
There was an update of geoip database. Alpine maintainer decided to delete update script from package because valid MaxMind license is now a must have for automated updates. Not sure if you can get goaccess working without properly updated geodb.